command_line/show.py

```diff
@@ -258,6 +258,9 @@ def show_experiments(experiments, show_scan_varying=False):
             text.append("Format class: %s" % format_class.__name__)
         if expt.identifier != "":
             text.append("Experiment identifier: %s" % expt.identifier)
+        template = expt.imageset.get_template()
```

Comment:
I may be wrong, but ISTR that `get_template()` only exisits for `ImageSequence`, not `ImageSet`?

Reply:
Yes, it seems you are right

Reply:
Even so the addition would be useful, could `try` / `except` or `hasattr` this one?

Reply:
Yep, done in 5ba4f498d8 :+1: